docs/static/breaking-changes.asciidoc

```diff
@@ -70,3 +70,32 @@ Starting with the 2.0 release of Logstash, the default value of the `filter_work
 plugins is half of the available CPU cores, instead of 1. This change increases parallelism in filter execution for
 resource-intensive filtering operations. You can continue to use the `-w` flag to manually set the value for this option,
 as in previous releases.
+
+
+[float]
+=== Ruby filter Changes
```

Comment:
Ruby Filter Changes (use title capitalization--by the way, the title of this topic should also be changed to say Breaking Changes)